routes/v1.js

```diff
@@ -766,6 +772,7 @@ api.declare({
   method:     'post',
   route:      '/task/:taskId/rerun',
   name:       'rerunTask',
+  stability:  base.API.stability.experimental,
```

Comment:
infrastructure retries don't trigger pulse messages. I think that was the piece missing.